src/decays/specializations/H/decay_H_to_AA.inc

```diff
@@ -29,5 +29,16 @@ double CLASSNAME::get_partial_width<Higgs, Photon, Photon>(
       );
    }
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
```

Comment:
See my comment on `static_cast<bool>`

Reply:
Done in c587517522b7b65208aa28eb3ac3b70a03a3ce92.